recipes/con-duct/meta.yaml

```
@@ -0,0 +1,51 @@
+{% set name = "con-duct" %}
+{% set version = "0.8.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.org/packages/source/{{ name[0] }}/{{ name }}/con_duct-{{ version }}.tar.gz
+  sha256: d7c1a5aeb0a6ee18803adef3ecdc61653529ab17846e8a14800be17a8bf50c68
+
+build:
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv --no-deps --no-build-isolation
+  number: 0
+
+requirements:
+  host:
+    - python >=3.9
+    - setuptools >=46.4.0
+    - pip
+  run:
+    - python >=3.9
+    - __glibc >=2.17  # Contstrain to linux
+    - matplotlib
+
+test:
+  imports:
+    - con_duct
+  commands:
+    - python -m pytest -vv test
+  requires:
+    - pytest
+    - pytest-cov
+    - matplotlib
```

```suggestion
    - matplotlib-base
```
per https://github.com/conda-forge/staged-recipes/pull/26894#issuecomment-2462644456 ?

---

https://github.com/con/duct/blob/main/src/con_duct/suite/plot.py#L59

IIRC using 'show()' does depend on the GUI backend (therefore full matplotlib), but it may be worth looking into a lighter weight backend for this simple case.